.github/workflows/validate_pr.yml

```diff
@@ -0,0 +1,38 @@
+name: Validate PR
+on:
+  pull_request:
+    paths-ignore:
+    - 'docs/**'
+    - '*.md'
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
```

Comment:
give this a try with `runs-on: self-hosted` We have 3 runners set up at the org level.